src/comm/LinkManager.cc

```diff
@@ -426,6 +426,9 @@ void LinkManager::loadLinkConfigurationList()
                                 pLink->setAutoConnect(autoConnect);
                                 pLink->loadSettings(settings, root);
                                 addConfiguration(pLink);
+                                if (pLink->isAutoConnect()) {
+                                    createConnectedLink(pLink);
```

Comment:
@dogmaphobic I think the positioning of this is ok. My only concern is whether this point in the code is maybe a little too early in boot time to start a link? Any comments?